docs/ref/utils.txt

```diff
@@ -590,7 +583,7 @@ escaping HTML.
 
     Returns the given text with ampersands, quotes and angle brackets encoded
     for use in HTML. The input is first passed through
-    :func:`~django.utils.encoding.force_text` and the output has
+    :func:`~django.utils.encoding.force_str` and the output has
```

Comment:
I feel we could get tickets saying this is inaccurate since the code still uses `force_text()`.